src/SimpleVideoCutter/MainForm.cs

```diff
@@ -615,21 +596,121 @@ private void StoreSettings()
             VideoCutterSettings.Instance.StoreSettings();
         }
 
-        private void EnqeueNewTask()
+
+        private void UnloadVideo()
+        {
+            if (vlcControl1.MediaPlayer != null)
+            {
+                vlcControl1.MediaPlayer.Stop();
+                vlcControl1.MediaPlayer.Media?.Dispose();
+                vlcControl1.MediaPlayer.Media = null;
+            }
+
+            if (videoViewHover.MediaPlayer != null)
+            {
+                videoViewHover.MediaPlayer.Stop();
+                videoViewHover.MediaPlayer.Media?.Dispose();
+                videoViewHover.MediaPlayer.Media = null;
+            }
+        }
+
+
+        private void ActionAfterTaskCompletion_ActionExecuting(object? sender, ActionExecutingEventArgs e)
+        {
+            if (e.Action is IActionRemovesOriginalFile && (e.Action as IActionRemovesOriginalFile)?.OriginalFilePath == this.fileBeingPlayed)
+            {
+                if (this.shouldNotifyIfCurrentFileIsBeingDeleted)
+                {
+                    DialogResult dialogResult = MessageBox.Show("As per the settings, the original file will be deleted after the cut was saved." +
+                        " Since you are currently playing this file, the program will skip to the next file first." +
```

Comment:
I recently cut about 30 files and enjoyed the jump to the next (I didn't care about the order, could also have been previous) file. Hm, it should be configurable.

Reply:
Yeah, in the end this is really a detail (whether to unload only or load another file immediately, next or prev). If there is a separate option to "close" file then I think it is not that important. 
On the other hand, it can always happen that there is no more files in the folder, thus we end up with "no file loaded" sitaution and this has to be supported.